cocos/platform/ohos/CCLogOhos.h

```diff
@@ -0,0 +1,121 @@
+#ifndef DISP_COMMON_H
```

Comment:
What's DISP_COMMON_H ?
Should it be `#ifndef __CC_LOG_OHOS_H__` ?

Reply:
ok, modified, will submit